<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Ручное управление</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Go </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Motor Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Go </string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Motor X</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Go </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Motor Z</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_10">
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leLineR"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_25">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Motor R</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Go To X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_8"/>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Go </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Go To Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_9"/>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Go </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Go To Z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_10"/>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Go </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Go To XY</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_11"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEdit_12"/>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>Go </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Pos X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Pos Z</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Pos Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_24">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Pos R</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_13"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>COM PORT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_7">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>COM4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>CONNECT</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Автоматическое управление</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QSplitter" name="splitter_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="btnDrawCircle">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Documents/0 Диплом/iconCircle.png</normaloff>../../Documents/0 Диплом/iconCircle.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QLineEdit" name="leCircleX"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leCircleY"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>160</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leCircleA"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leCircleB"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="btnDrawRect">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Documents/0 Диплом/iconRectangle.png</normaloff>../../Documents/0 Диплом/iconRectangle.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leRectX"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>L1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="leRectL1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leRectY"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>L2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="leRectL2"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="btnDrawLine">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Documents/0 Диплом/iconLine.png</normaloff>../../Documents/0 Диплом/iconLine.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>X1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leLineX1"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>X2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="leLineX2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Y1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leLineY1"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Y2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="leLineY2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GoToPos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POS</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
